.storybook/preview.ts

```diff
@@ -1,15 +1,43 @@
-import type { Preview } from "@storybook/react";
+import type { Preview } from '@storybook/react';
+
+import { withThemeByClassName } from '@storybook/addon-styling';
+
+/* TODO: update import to your tailwind styles file. If you're using Angular, inject this through your angular.json config instead */
+import '../app/globals.css';
+
+/* TODO: update import to your tailwind styles file. If you're using Angular, inject this through your angular.json config instead */
+//import '../src/index.css';
 
 const preview: Preview = {
   parameters: {
-    actions: { argTypesRegex: "^on[A-Z].*" },
+    actions: { argTypesRegex: '^on[A-Z].*' },
     controls: {
       matchers: {
         color: /(background|color)$/i,
         date: /Date$/,
       },
     },
   },
+
+  decorators: [
+    // Adds theme switching support.
+    // NOTE: requires setting "darkMode" to "class" in your tailwind config
+    withThemeByClassName({
```

Comment:
It seems like duplicated withThemeByClassName here? Can we remove one of them?

Reply:
yes, duplicate removed